### PR TITLE
move 'removeLastPoint' to GeometryBuffer

### PR DESCRIPTION
**Pull request by Gustl22**

### CODE REVIEW

File: vtm/src/org/oscim/core/GeometryBuffer.java [hunk 1]
```diff
@@ -468,6 +468,22 @@ public float isClockwise() {
...
+    /**
+     * Remove the last point
+     */
+    public void removeLastPoint() {
+        if (isTris()) {
+            int indexSize = 0;
+            while (indexSize < index.length && index[indexSize] != -1) {
+                indexSize++;
+            }
+            index[indexSize - 3] = -1;
```
Comment (devemux86) [inline vtm/src/org/oscim/core/GeometryBuffer.java]: Only `index` needs changes?

Comment (Gustl22) [reply vtm/src/org/oscim/core/GeometryBuffer.java]: Yes, points could be referenced by lower indices, so wouldn't be good to remove them.

**Comment by Gustl22** [comment]
Hold on, I'll add some tests.

**Comment by Gustl22** [comment]
Removing last point in a TRIS is not useful / difficult to implement, as faces have to be recalculated. The previous implementation removed the last `face` (tris) of buffer.

**Comment by devemux86** [comment]
Merged via fb8e6dab20a5afc9ca23e3e42f8b605250745dc7.

**Pull request closed without merging**